tasks/components/RoomOverview.tsx

```diff
@@ -3,26 +3,54 @@ import { Span } from '@helpwave/common/components/Span'
 import { useContext, useEffect, useRef, useState } from 'react'
 import type { RoomOverviewDTO } from '@helpwave/api-services/types/tasks/room'
 import type { BedMinimalDTO } from '@helpwave/api-services/types/tasks/bed'
-import type { PatientDTO } from '@helpwave/api-services/types/tasks/patient'
+import type { Gender, PatientDTO } from '@helpwave/api-services/types/tasks/patient'
 import { emptyPatient } from '@helpwave/api-services/types/tasks/patient'
+import type { AppColor } from '@helpwave/common/twind/config'
+import type { Languages } from '@helpwave/common/hooks/useLanguage'
+import type { PropsForTranslation } from '@helpwave/common/hooks/useTranslation'
+import { useTranslation } from '@helpwave/common/hooks/useTranslation'
 import { BedCard } from './cards/BedCard'
 import { PatientCard } from './cards/PatientCard'
 import { Droppable, Draggable } from './dnd-kit-instances/patients'
 import { DragCard } from './cards/DragCard'
 import { WardOverviewContext } from '@/pages/ward/[wardId]'
 
+type RoomOverviewTranslation = Record<Gender, string>
+
+const defaultRoomOverviewTranslations: Record<Languages, RoomOverviewTranslation> = {
+  en: {
+    male: 'M',
+    female: 'F',
+    diverse: 'D',
+  },
+  de: {
+    male: 'M',
+    female: 'F',
+    diverse: 'D',
+  }
+}
+
 export type RoomOverviewProps = {
-  room: RoomOverviewDTO
+  room: RoomOverviewDTO,
+  gender?: Gender // TODO get from room later on
 }
 
 /**
  * A component to show all beds and patients within a room in a ward
  */
-export const RoomOverview = ({ room }: RoomOverviewProps) => {
+export const RoomOverview = ({ room, gender = 'male', overwriteTranslation }: PropsForTranslation<RoomOverviewTranslation, RoomOverviewProps>) => {
```

Comment:
Why do you pass the gender as a prop on a room? The patient has a gender, not the room. Therefore the bed needs this indication.